### PR TITLE
2018 CVE Rejections

### PR DESCRIPTION
**Pull request by lenpsirt**
Rejecting unused 2018 CVEs

### CODE REVIEW

File: 2018/16xxx/CVE-2018-16107.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16107.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16110.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16110.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16104.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16104.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16113.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16113.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16108.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16108.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16109.json [hunk 6]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16109.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16111.json [hunk 7]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16111.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16106.json [hunk 8]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16106.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16100.json [hunk 9]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16100.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16101.json [hunk 10]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16101.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16103.json [hunk 11]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16103.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16105.json [hunk 12]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16105.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16112.json [hunk 13]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16112.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16099.json [hunk 14]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16099.json]: 'REJECTED' is not a supported STATE!

File: 2018/16xxx/CVE-2018-16102.json [hunk 15]
```diff
@@ -2,7 +2,7 @@
    "CVE_data_meta" : {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16102.json]: 'REJECTED' is not a supported STATE!

**Comment by lenpsirt** [comment]
Changed State from REJECTED to REJECT

File: 2018/16xxx/CVE-2018-16101.json (hunk 10 above)
Comment (lenpsirt) [reply 2018/16xxx/CVE-2018-16101.json]: Updated state to REJECT on all failures.

**Comment by cve-team** [comment]
Thanks for your submission. This information has been processed and the changes should be reflected on cve.mitre.org within an hour or two.

**Pull request merged**